spec/services/idv/session_spec.rb

```diff
@@ -47,27 +47,24 @@
     end
   end
 
-  describe '#method_missing' do
-    it 'disallows un-supported attributes' do
+  describe 'attribute methods' do
+    it 'disallows un-supported setters' do
       expect { subject.foo = 'bar' }.to raise_error NoMethodError
     end
 
-    it 'allows supported attributes' do
+    it 'allows using supported setters and getters' do
       Idv::Session::VALID_SESSION_ATTRIBUTES.each do |attr|
-        subject.send attr, 'foo'
```

Comment:
So before calling `idv_session.address_edited('foo')` would act as a setter.

That is very uncommon! I made a call to remove that, now it will throw an argument error because now reader and writer are two separate methods.

Open to changing this back if we are worried, but I expect our test suite would catch any lingering use of that style

Reply:
I noticed this too, glad to see a fix

Reply:
fixed two sneaky-setter usages in f2426a2ec3d75703995cd8004f387db2cb34bb0c